lib/timescaledb/model/peer_metrics.ex

```diff
@@ -10,19 +10,19 @@ defmodule Membrane.RTC.Engine.TimescaleDB.Model.PeerMetrics do
   alias Membrane.RTC.Engine.TimescaleDB.Model.TrackMetrics
 
   @type t :: %__MODULE__{
-          id: integer(),
-          peer_id: String.t(),
+          id: integer() | nil,
+          peer_id: String.t() | nil,
+          room_id: String.t() | nil,
           "ice.binding_requests_received": non_neg_integer() | nil,
           "ice.binding_responses_sent": non_neg_integer() | nil,
           "ice.bytes_received": non_neg_integer() | nil,
           "ice.bytes_sent": non_neg_integer() | nil,
           "ice.packets_received": non_neg_integer() | nil,
           "ice.packets_sent": non_neg_integer() | nil,
-          tracks_metrics: [TrackMetrics] | nil,
-          inserted_at: DateTime.t()
+          tracks_metrics: [TrackMetrics.t()] | Ecto.Association.NotLoaded.t(),
+          inserted_at: DateTime.t() | nil
```

Comment:
This is `nil` only if not yet inserted, right? If so, then PeerMetrics might have `nil` here too

Reply:
Value stored under `:track_metrics` key in newly created struct is `#Ecto.Association.NotLoaded<>`, not `nil`

Reply:
The comment refers to line 23 with `:inserted_at` key